### PR TITLE
Cosmatic Drip 1

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Adds a bunch of clothes, including departmental and universal options (this will take a bit)

## Why / Balance
There is a criminal lack of options outside of the randomly generated colourful uniforms

## Technical details
A bunch of new items

## Media
image
imageimage
imageimage
image
imageimage

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
🆑
add: Added three new security uniforms and one paramedic uniform in both suit and skirt variants.
add: Senior beret for security
add: Two cloaks, stay bright and warm up
add: Three heavy coats for cold nights on Europa
add: Engineering sweaters, don't mind the jorts
add: Cargo pants and a brown gorka
add: Three high-vis vests to be easily seen
add: Comfortable clothes for Doctors and Physicians
add: A new coat for central command officers

### CODE REVIEW

**Comment by dffdff2423** [comment]
This has been sitting as a draft for quite some time. If what you have now is good to merge can you open this? You can always PR the rest of it later.

**Comment by KittenColony** [comment]
I still have several things I'd like to add first, I've just gotten stuck on things

**Comment by KittenColony** [comment]
my depression is manic but my drip is ecstatic...

**Comment by KittenColony** [comment]
failure unrelated for once

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 1]
```diff
@@ -33,3 +33,130 @@
...
+  id: ClothingOuterCoatCargoBomber
+  name: cargo gorka
+  description: A heavy canvas gorka for cold shifts in Cargo.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:90]: You forgot to include the standard slash protection that winter coats get.

Any reason to not inherit from both ClothingOuterStorageFoldableBase and ClothingOuterWinterCoat?

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 2]
```diff
@@ -33,3 +33,130 @@
...
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterCoatYellowHiVis]
+  id: ClothingOuterCoatYellowHiVisOpened
+  name: emergency jacket
+
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:129]: Same as above

**Comment by dffdff2423** [comment]
No, it is an arbitrage thing again. You need to up the price of a CrateVendingMachineRestockQuickThreads

File: Resources/Prototypes/Loadouts/Jobs/Cargo/cargo_technician.yml [hunk 3]
```diff
@@ -65,6 +65,15 @@
...
 
+- type: loadout # CD
+  id: CargoBomber
+  equipment: CargoBomber
+
+- type: startingGear # CD
+  id: CargoBomber
+  equipment:
+    outerClothing: ClothingOuterCoatCargoBomber
+
```
Comment (dffdff2423) [inline Resources/Prototypes/Loadouts/Jobs/Cargo/cargo_technician.yml:76]: These can be in a separate file under _CD

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 4]
```diff
@@ -33,3 +33,130 @@
...
+  id: ClothingOuterCoatYellowHiVis
+  name: emergency jacket
+  description: A somewhat heavy jacket made of fluorescent fabric. This one is neon yellow.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/yellow_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/yellow_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:121]: Same issue as above, although I am not sure that a HiVis jacket should have temperature protection?

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 5]
```diff
@@ -33,3 +33,130 @@
...
+  id: ClothingOuterCoatBlueHiVis
+  name: EMT jacket
+  description: A somewhat heavy jacket made of fluorescent fabric. This one is neon blue.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/blue_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/blue_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:157]: Same as above, also not sure if EMT clothes should have temperature protection.

File: Resources/Prototypes/_CD/Entities/Clothing/Uniforms/fabric_wrap.yml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+#          topsheet-equipped-INNERCLOTHING: Sixteen
+#        middle:
+#          middlesheet-equipped-INNERCLOTHING: Sixteen
+#        bottom:
+#          bottomsheet-equipped-INNERCLOTHING: Sixteen
+#        trim:
+#          trimprimary-equipped-INNERCLOTHING: Sixteen
+#        trim2::
+#          trimsecondary-equipped-INNERCLOTHING: Sixteen
+# LMAO FUCK THIS AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/Uniforms/fabric_wrap.yml:64]: Can you remove the commented out code from the PR? You can keep it around locally, it is typically not best practice to commit commented out code (that is what a VCS is for).

**Comment by dffdff2423** [comment]
That is not going to be enough, you need at least 4130 according to the test. Just set it to 4500 to be safe.

File: Resources/Textures/_CD/Clothing/Neck/Cloaks/illuminated_cloak.rsi/meta.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by icestormthedragon (661569494728048671) on Discord",
```
Comment (dffdff2423) [inline Resources/Textures/_CD/Clothing/Neck/Cloaks/illuminated_cloak.rsi/meta.json:4]: Just highlighting this so we remember to give Ice a contrib role.

File: Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json [hunk 8]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC0-1.0",
+  "copyright": "Base made by Psychpsyo, Altered into current form by KittenColony",
```
Comment (dffdff2423) [inline Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json:4]: Do you have a source for this? Or was it just provided to you privately?

**Review by dffdff2423 — CHANGES_REQUESTED** [review]
Art looks fine to me, but I am not exactly somebody with a ton of artistic talent.

File: Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json (hunk 8 above)
Comment (dffdff2423) [reply Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json:4]: This also goes for all of the other files credited to Psychpsyo

Comment (KittenColony) [reply Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json:4]: space asshole jacket

Comment (dffdff2423) [reply Resources/Textures/_CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi/meta.json:4]: Can you add that to the credits?

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml (hunk 2 above)
Comment (dffdff2423) [reply Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:129]: Tempature protection for HiVis coats discussed here: https://discord.com/channels/1163352113162768436/1163352113859002400/1265796891187085334

It makes sense with the reference image.

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml (hunk 1 above)
Comment (KittenColony) [reply Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:90]: it can open and close, winter coat doesn't have that function

Comment (dffdff2423) [reply Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:90]: You can inherit from both.

eg: 
```
parent: [ ClothingOuterStorageFoldableBase, ClothingOuterWinterCoat ]
```

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 9]
```diff
@@ -119,6 +119,11 @@
     sprite: _CD/Clothing/OuterClothing/Coats/Highvis/yellow_bomber.rsi
   - type: TemperatureProtection
     coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.90
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:126]: If you inherit from ClothingOuterWinterCoat this is included automatically.

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml (hunk 1 above)
Comment (KittenColony) [reply Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:90]: damn you for having good ideas, give me a moment

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml (hunk 2 above)
Comment (dffdff2423) [reply Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:125]: This still needs to inherit from ClothingOuterCoatYellowHiVis. Assuming you want the damage resistances.

File: Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml [hunk 10]
```diff
@@ -33,3 +33,122 @@
...
+  description: A comfortable green coat worn by CentCom officers. The armour has been removed in favour of more pocket space.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/central_officer.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/central_officer.rsi
+
+# High-Vis jackets
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterWinterCoat]
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:109]: ClothingOuterStoargeFodableBase**Opened**

Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml:109]: typo: ClothingOuterStorageFoldableBase**Opened**

**Review by dffdff2423 — APPROVED** [review]
I am going to say LGTM. I did a smoke test in game and the few items I tested seemed to work as expected.

**Review by Nairodian — APPROVED** [review]
Fine by me sprite-wise. I might go into finer detail after it this gets merged and I see how it looks in proper use by players.

**Comment by he1acdvv** [comment]
Legendary pull.

**Comment by KittenColony** [comment]
note, blue highvis is glitchy, need to fix

**Comment by he1acdvv** [comment]
Maybe make the bright white slightly darker? (akin to the white on cargo gorkas)

**Pull request merged**